vimeo-networking/src/main/java/com/vimeo/networking/model/search/SuggestionResponse.java

```diff
@@ -0,0 +1,68 @@
+package com.vimeo.networking.model.search;
+
+import com.google.gson.annotations.SerializedName;
+import com.vimeo.stag.UseStag;
+import com.vimeo.stag.UseStag.FieldOption;
+
+import org.jetbrains.annotations.Nullable;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+
+/**
+ * A response sent from the API for search/autocomplete suggestions.
+ * <p>
+ * Created by zetterstromk on 2/21/17.
+ */
+@UseStag(FieldOption.SERIALIZED_NAME)
+public class SuggestionResponse implements Serializable {
```

Comment:
It's too bad that this response can't extend from `BaseResponseList` but I guess it's fundamentally a different type of data (i.e. not pageable)